### PR TITLE
QE: Add missing Leap Micro 5.5 entries to `constants.rb`

### PR DESCRIPTION
**Pull request by nodeg**
## What does this PR change?

This PR

- adds the missing Leap Micro 5.5 entries to our constants
- adds some debugging output for the bootstrap creation step in the BV. With this, we spot a missing entry in `constants.rb` much easier/quicker.

Found and also tested in the Uyuni BV.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes
- [x] **DONE**

## Test coverage
- Cucumber tests were edited
- [x] **DONE**

## Links

Issue(s): #
Ports(s): # 

- [ ] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: testsuite/features/step_definitions/command_steps.rb [hunk 1]
```diff
@@ -1054,6 +1054,10 @@
   parent_channel = PARENT_CHANNEL_LABEL_TO_SYNC_BY_BASE_CHANNEL[product][base_channel]
   get_target('server').wait_while_process_running('mgr-create-bootstrap-repo')
 
+  $stdout.puts "base_channel: #{base_channel}"
+  $stdout.puts "channel: #{channel}"
+  $stdout.puts "parent_channel: #{parent_channel}"
+
```
Comment (Bischoff) [inline testsuite/features/step_definitions/command_steps.rb:1060]: Don't we want to use `log` function or something alike?

Or did you just forget to remove these tests?

**Review by Bischoff — COMMENTED** [review]
see question

other than that lgtm

File: testsuite/features/step_definitions/command_steps.rb (hunk 1 above)
Comment (nodeg) [reply testsuite/features/step_definitions/command_steps.rb:1060]: From what I can see, we use both a lot. So I am also fine with changing that to `log " "`.

Comment (Bischoff) [reply testsuite/features/step_definitions/command_steps.rb:1060]: I think it's Oscar who pushes for using log(). I am fine with that policy.

Comment (nodeg) [reply testsuite/features/step_definitions/command_steps.rb:1060]: Ok, then I am switching, too.

Comment (nodeg) [reply testsuite/features/step_definitions/command_steps.rb:1060]: Fixed in new commit.

**Review by Bischoff — APPROVED** [review]

**Review by NamelessOne91 — APPROVED** [review]
LGTM, thanks !

**Pull request merged**